Front-End/challenge2/script.js

```diff
@@ -5,3 +5,97 @@ TO-DO:
 - The calculator should be completely functional
 
 */
+const zero = document.getElementById("zero");
```

Comment:
how could you change this code to get the elements you need without adding a specific selector for each element? what would happen if instead of 9 numbers you have 100 numbers? how would you resolve this?

Reply:
using document.querySelectorAll("button")
and then filtering the values ​​that are numbers and those that are operators